src/WebimpressCodingStandard/ruleset.xml

```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0"?>
-<ruleset name="Webimpress Coding Standard"
+<ruleset name="WebimpressCodingStandard"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="../../vendor/squizlabs/php_codesniffer/phpcs.xsd">
     <description>Webimpress Coding Standard</description>
 
-    <!-- display progress -->
-    <arg value="p"/>
-    <arg name="colors"/>
```

Comment:
Why these are removed?
I wanted to have enabled progress and colours by default.

Reply:
Because command line options should be in the user's phpcs.xml, not in the coding standard.